### DISCUSSION TITLE
UI bug with hovering

### DISCUSSION THREAD

**Discussion by yairEO** [community/community#14382] [Code Security] [answered]
1. There's a delay on the hover row for some reason. The faded-out row show not have any delay on hover. only on hover-out. 
2. The faded-out row still acts is if hovered if the mouse leaves from the sides
3. The information row vanishes (see seconds 5-6 of below video)

https://user-images.githubusercontent.com/845031/161962820-1b9b799e-48fc-40ba-acac-be2fa3f9495b.mp4

**Comment by jhutchings1** [comment, marked as answer]
Thanks for your feedback! The hover element is on the title to the PR, not on the whole table row, so the behavior you saw is working as intended. Sorry for the confusion!

**Reply by yairEO** [reply]
OK but what about the actual bug with the hover? Do you agree there's a bug there?

**Reply by jhutchings1** [reply]
Can you provide more repro steps? In talking with design about this and we're not sure which UX you've recorded

**Reply by yairEO** [reply]
`Pull Requests` page - [here's an example](https://github.com/open-amdocs/webrix/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed). Hover (with mouse) a faded-out item and move the mouse to the left, outside of the table. It will still be looked as "hovered"
